roles/ansible-test-splitter/files/test_split_targets.py

```diff
@@ -34,6 +58,16 @@ def test_is_slow():
     assert is_slow(["noslow", "# reason: noslow"]) is False
 
 
+@patch.object(PosixPath, "glob", MagicMock(return_value=target_dir))
+def test_get_all_targets():
+    assert get_all_targets("somewhere") == {
+        "t1": ["cloud/aws", "slow"],
+        "t2": ["t1"],
+        "t3": ["cloud/aws", "", "disabled"],
+    }
+
+
+
```

Comment:
```suggestion
```

Reply:
I extended patch to fix the problem and run the black formatter through tox (and the CI).